api-extractor/src/definitions/ApiPackage.ts

```diff
@@ -7,18 +7,38 @@ import ApiEnum from './ApiEnum';
 import ApiFunction from './ApiFunction';
 import { IApiItemOptions } from './ApiItem';
 import ApiItemContainer from './ApiItemContainer';
+import TypeScriptHelpers from '../TypeScriptHelpers';
 
 /**
   * This class is part of the ApiItem abstract syntax tree.  It represents the top-level
   * exports for an Rush package.  This object acts as the root of the Analyzer's tree.
   */
 export default class ApiPackage extends ApiItemContainer {
-  constructor(analyzer: Analyzer, rootFileSymbol: ts.Symbol) {
-    super({
+  private static _getOptions(analyzer: Analyzer, rootFileSymbol: ts.Symbol): IApiItemOptions {
+    let statement: ts.VariableStatement;
+    let foundDescription: ts.Node = undefined;
+    for (const statementNode of (rootFileSymbol.declarations[0] as ts.SourceFile).statements) {
+      if (statementNode.kind === ts.SyntaxKind.VariableStatement) {
+        statement = statementNode as ts.VariableStatement;
```

Comment:
The second link was very useful. 

"If we have a value that has a union type, we can only access members that are common to all types in the union."
"If a value has the type A | B, we only know for certain that it has members that both A and B have. In this example, Bird has a member named fly. We can’t be sure whether a variable typed as Bird | Fish has a fly method. If the variable is really a Fish at runtime, then calling pet.fly() will fail."